cbdr/cortex
# COR-698: validation fixes

**Purpose:**
The purpose of this PR is to fix up and improve the Validations system for Custom Content Cortex before the wider scale release to ensure the validations both work and are helpful/user-friendly. 

> As a writer, I want clearer expectations of what each field needs to be fulfilled in my blog creation state.

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/COR-698

**Changes:**
* Changes to setup
  * Reset / Reseed Cortex

* Architectural changes
  * Minor Updates to Cortex-Plugins-Core

* Migrations
  * N/A
  
* Library changes
  * Cortex-Plugins-Core: https://github.com/cortex-cms/cortex-plugins-core/pull/52

* Side effects
  * N/A

**Screenshots**
* Before
N/A

* After
<img width="398" alt="screen shot 2017-03-13 at 2 34 37 pm" src="https://cloud.githubusercontent.com/assets/8419757/23871883/3c06404c-07fa-11e7-8912-ed77b3d6d009.png">

**QA Links:**
N/A

**How to Verify These Changes**
* Specific pages to visit
  * New Employer Blog

* Steps to take
  * Create an Employer Blog half assedly
  * See Errors :+1:

* Responsive considerations
  * N/A

**Relevant PRs/Dependencies:**
https://github.com/cortex-cms/cortex-plugins-core/pull/52

**Additional Information**
N/A
